src/main/config/parameter_group_ids.h

```diff
@@ -66,6 +66,8 @@
 #define PG_MSP_SERVER_CONFIG 48
 #define PG_VOLTAGE_METER_CONFIG 49
 #define PG_AMPERAGE_METER_CONFIG 50
+#define PG_MOTOR_CONFIG 51
```

Comment:
@martinbudden yes, i think @ledvinap is probably right, more people use motors than servos so having automatic upgrade is probably a good idea.  call the dummy value 'reserved' or something similar.   

We would encounter an issue in the future if we were to use 'reserved' for something else without it being sanitized/reset.

Perhaps place a comment to remind us that if the value is used migrations should ignore any value stored in the 'reserved' field.